libc/src/math/generic/exp.cpp

```diff
@@ -199,7 +199,7 @@ double set_exceptional(double x) {
       return x;
 
     if (fputil::quick_get_round() == FE_UPWARD)
-      return static_cast<double>(FPBits(FPBits::MIN_SUBNORMAL));
+      return FPBits::min_denormal();
```

Comment:
[nit] We probably want to be consistent with sub vs denormal.

Reply:
Thx for pointing this out. Let's do this in a subsequent patch.